python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -461,11 +499,12 @@ async def telemetry_callback(self, client):
             self._tracking_started_telemetry_counter -= 1
         await self.evt_summaryState.set_write(summaryState=self.summary_state)
 
-        # Strangely telemetry.state and enabled_substate are floats
-        # from the controller. But they should only have integer value,
+        # Strangely telemetry.state, fault_substate, and enabled_substate are
+        # floats from the controller. But they should only have integer value,
         # so I output them as integers.
         await self.evt_controllerState.set_write(
             controllerState=int(client.telemetry.state),
+            faultSubstate=int(client.telemetry.fault_substate),
```

Comment:
Consider adding some level of backward compatibility with previous versions of xml. I'd suggest doing something like this:

```
controller_state_data = dict(
    controllerState=int(client.telemetry.state),
    enabledSubstate=int(client.telemetry.enabled_substate),
    applicationStatus=client.telemetry.application_status,    
)

if  hasattr(self.evt_controllerState.DataType(), "faultSubstate"):
    controller_state_data["faultSubstate"] = int(client.telemetry.fault_substate)

await self.evt_controllerState.set_write(
    **controller_state_data
)
```

Maybe add a `TODO` to remove backward compatibility.

Reply:
Fixed in the commit: [f03b120](https://github.com/lsst-ts/ts_mtrotator/pull/82/commits/f03b120f7da830090949ea0b1a365e87fd3689b1). Thanks!